widgets/features/features.php

```diff
@@ -253,6 +253,7 @@ function get_less_variables( $instance ) {
 			}
 		}
 
+		$less_vars['responsive_breakpoint'] = siteorigin_panels_setting('mobile-width') . 'px';
```

Comment:
We just need to deal with the condition where Page Builder isn't being used. This might be a rare case now, but it'll become more common with the work we've done on getting Widgets Bundle to play-nice with more page builder plugins.

Right now, this will cause a fatal error if Page Builder isn't installed.